docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -37,13 +37,48 @@ Custom Search Attributes of the `Text` type cannot be used in **ORDER BY** claus
 
 ### Partial string match
 
-The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
+#### Text
```

Comment:
Usually we try not to put a header right after another header.
Perhaps some contextualizing copy between them?